src/mongocrypt-key.c

```diff
@@ -119,6 +128,7 @@ _mongocrypt_key_alt_name_from_iter (const bson_iter_t *iter_in,
       }
 
       tmp = _mongocrypt_key_alt_name_new (bson_iter_value (&iter));
+      BSON_ASSERT (tmp);
```

Comment:
Redundant assertion: `_mongocrypt_key_alt_name_new` (`bson_malloc0`) is guaranteed to return a not-null pointer.

Reply:
Removed.  Change coming in next commit.